### ISSUE TITLE
Disable specific or multiple dates

### ISSUE THREAD

**Issue by ittikorns** [stephy/CalendarPicker#67]
The idea is to add disabledDates prop with list of date in array format which will become unable to choose from the picker.

I've created a PR #66  for this.

**Example code**
```
const disabledDates = [new Date('10 May, 2017'), new Date('12 May, 2017'), new Date('15 May, 2017')];

<CalendarPicker
...
disabledDates={disabledDates}
...
/>
```

**Screenshots**
<img width="369" alt="screen shot 2560-05-04 at 11 28 58 pm" src="https://cloud.githubusercontent.com/assets/1611115/25714224/7b781bec-3121-11e7-8621-1ff20e490a6c.png">

**Comment by ghitatomy** [comment]
Can you please fix this?

**Comment by peacechen** [comment]
This has been implemented in #104 and merged into master.
